app/views/themes/reshare_show/hyrax/base/_attribute_rows.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= presenter.attribute_to_html(:admin_note, html_dl: true) if presenter.editor? %>
```

Comment:
do you need this file? Is there custom metadata for ReShare? Or does this have the same items as the base attribute_rows?

Reply:
It has the same items as base attribute rows, but they are not rendering as faceted as in the base attribute rows since users should not be able to go to any hyku search pages